ql/src/test/queries/clientpositive/execute_query_string.q

```diff
@@ -0,0 +1,3 @@
+create table prepareexecute(a int, b int);
+prepare pcount from select count(*) from prepareexecute where a > ?;
+execute pcount using '200';
```

Comment:
The new test is redundant since the same query pattern appears in other .q files and the effects of the changes are clearly visible in the .q.out changes.

Reply:
Yes you are right, I will remove the new q file